unit-tests/py/rspy/devices.py

```diff
@@ -214,7 +215,10 @@ def query( monitor_changes = True ):
     if acroname:
         if not acroname.hub:
             acroname.connect()  # MAY THROW!
-            acroname.enable_ports( sleep_on_change = 5 )  # make sure all connected!
+
+            acroname.disable_ports( sleep_on_change = 5 )
```

Comment:
Maybe we can wait until all ports disabled, instead?
Might need to take special care with D457, but should be better.